### ISSUE TITLE
NVDA fails to properly read list of added folders in the properties window for a "library" in Windows

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#8383]
### Steps to reproduce:
1. Go to the "Libraries" folder on the computer.
2. Select one of the libraries and open its properties.
3. In the "Libraries locations" list box, use the list item reading commands of NVDA to read each item in the list.

### Expected behavior:
NVDA should read each list item, at least as it is displayed on the screen.

### Actual behavior:
NVDA does not read any item and just plays its error sound.

### System configuration:
- Operating system: Windows 10 Pro version 1803 (build 17134.81), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-15211,8a5d09f0.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

### Other questions:
1. Does the issue still occur after restarting your PC: Yes, it does.
2. Have you tried any other versions of NVDA: Yes, NVDA 2018.1.1 and with it the result is much better, but not ideal. With it, the list items are read, but with a string of random characters before each list item.

### Additional notes
I haven't used this functionality of Windows in a long time, but when I did, it was working OK. On inspection of the NVDA log file that I'll attach here, it seams that the new version of eSpeak is partially to blame. If I use another synthesizer, the behavior is the one described in point number 2 in the "Other questions" section above. Even so however, the behavior from point number 2 in the "Other questions" section is not the expected behavior either, so in both cases we have a bug here.
[NVDA_2018-06-10_12-46-02.log](https://github.com/nvaccess/nvda/files/2087707/NVDA_2018-06-10_12-46-02.log)

**Comment by k-kolev1985** [comment]
@Adriani90 Yes, I can still reproduce the issue described above.

**Comment by Adriani90** [comment]
@k-kolev1985 are you still having this issue in NVDA 2019.1.1 on Windows 10 1809 Update?

**Comment by Adriani90** [comment]
@k-kolev1985 I am testing with last Alpha in Windows 10 1909 Update, there are no errors. I can navigate the lsit with arrow keys or ctrl+alt+arrow keys. However the part with strange numbers and letters before the location name is read although it is not recognized by OCR. So that means it is actually not visible. However, this seems part of the object model, you can check with object navigation. Could you please report this to Microsoft as well? I think they need to fix it.

**Comment by k-kolev1985** [comment]
@Adriani90 Yes, I can report it to Microsoft, but more technical details about this from more knowledgeable people would be welcome, so the report is more valid. Because now the problem occurs with NVDA, but not with Narrator and based on that Microsoft may say that the problem is in NVDA and not in Windows. But if I report it with more technical details as proof, it will be harder for them to reject the report.

**Comment by Adriani90** [comment]
maybe @josephsl can help here with some technical details for this issue.

**Comment by Adriani90** [comment]
@k-kolev1985 I cannot reproduce this issue anymore in NVDA last alpha version and Windows 11.

Are you still having this issue?

**Comment by k-kolev1985** [comment]
@Adriani90 Yes, I can still partially reproduce the issue. Although now even with eSpeak each list item is read, the combination of letters and numbers before each folder name is still present. And this does not occur with Narrator - it reads only the folder's name.

**Comment by Adriani90** [comment]
@josephsl it seems the locations in the tree view when opening the properties window for a library contain very strange child elements with very long combination of numbers and letters. Is it possible for NVDA to ignore those child elements, same as Narator does? Maybe to move them to the non simplified object navigation?
cc: @CyrilleB79
